src/grpc/dp_grpc_impl.c

```diff
@@ -59,22 +59,24 @@ static __rte_always_inline void dp_generate_underlay_ipv6(uint8_t route[DP_VNF_I
 
 static int dp_create_vnf_route(uint8_t ul_addr6[DP_VNF_IPV6_ADDR_SIZE] /* out */,
 							   enum dp_vnf_type type, uint32_t vni, const struct dp_port *port,
-							   uint32_t prefix_ip, uint8_t prefix_len)
+							   struct dp_ip_address *pfx_ip, uint8_t prefix_len)
 {
 	dp_generate_underlay_ipv6(ul_addr6);
-	return dp_add_vnf(ul_addr6, type, port->port_id, vni, prefix_ip, prefix_len);
+	return dp_add_vnf(ul_addr6, type, port->port_id, vni, pfx_ip, prefix_len);
 }
 
 static int dp_process_create_lb(struct dp_grpc_responder *responder)
 {
 	struct dpgrpc_lb *request = &responder->request.add_lb;
 	struct dpgrpc_ul_addr *reply = dp_grpc_single_reply(responder);
+	struct dp_ip_address pfx_ip;
 
 	uint8_t ul_addr6[DP_VNF_IPV6_ADDR_SIZE];
 	int ret = DP_GRPC_OK;
 
-	if (request->addr.ip_type == RTE_ETHER_TYPE_IPV4) {
-		if (DP_FAILED(dp_create_vnf_route(ul_addr6, DP_VNF_TYPE_LB, request->vni, dp_get_pf0(), 0, 0))) {
+	memset(&pfx_ip, 0, sizeof(pfx_ip));
```

Comment:
I think the intention is to create an equivalent of `0.0.0.0`, but this `pfx_ip` is invalid, because in most places in the code you check the `ip_type` to be IPv4 or IPv6 and fail otherwise.

Reply:
Replaced it with

> 	struct dp_ip_address pfx_ip = {
		.ip_type = request->addr.ip_type
	};

for every occurrence.